src/workers/postTranslated.ts

```diff
@@ -0,0 +1,30 @@
+import { TypedWorker } from './worker';
+import { Post } from '../entity';
+import { validLanguages } from '../types';
+import { logger } from '../logger';
+
+export const postTranslated: TypedWorker<'kvasir.post-translated'> = {
+  subscription: 'api.post-translated',
+  handler: async (message, con) => {
+    const { id, translations, language } = message.data;
+
+    if (!validLanguages.includes(language)) {
+      logger.error({ id, language }, '[postTranslated]: Invalid language');
```

Comment:
would we not throw so then we can update languages and it auto reprocesses?

Reply:
Could lead to a massive backlog of unprocessed languages. The same check happens in Kvasir too, so in API it's more of a safeguard.

Reply:
Yeah thats true of course. How would we add a support for new language for example, what thats updated when?

Reply:
I guess we maybe could store it in Growthbook, then Kvasir can also read from that?

Reply:
Not a bad idea yes, because now we would have to update on both places?

Reply:
Yeah, need to update both places. And technically front-end too so it's a dropdown option.